vendor-extra/ViewsBundle/src/Views/TemplateView.php

```diff
@@ -65,7 +65,7 @@ public function withContext(array $context) : TemplateView
     {
         $view = clone $this;
 
-        $view->context = array_replace_recursive($view->context, $context);
+        $view->context = array_replace($view->context, $context);
```

Comment:
`array_replace_recursive()` was causing problems as the list wasn't being replaced entirely.